components/Snackbar/examples/SnackbarSimpleExample.m

```diff
@@ -157,6 +157,14 @@ - (void)showCustomizedSnackbar:(id)sender {
 - (void)showDecustomizedSnackbar:(id)sender {
   [MDCSnackbarMessageView appearance].messageFont = nil;
   [MDCSnackbarMessageView appearance].buttonFont = nil;
+
+  // Setting back to the default colors as defined in MDCSnackbarMessageView.h.
+  [[MDCSnackbarMessageView appearance] setButtonTitleColor:[UIColor colorWithWhite:1 alpha:0.6f]
+                                                  forState:UIControlStateNormal];
+  [[MDCSnackbarMessageView appearance] setButtonTitleColor:UIColor.whiteColor
+                                                  forState:UIControlStateHighlighted];
+  [MDCSnackbarMessageView appearance].messageTextColor = UIColor.whiteColor;
+
```

Comment:
I would expect all of these properties to be null-resettable similar to the .Font properties above.
Is that possible?